doc/doxygen/references.bib

```diff
@@ -950,6 +1099,7 @@ @TechReport{Saad1991
   Address                  = {University of Minnesota},
   Number                   = {91-279},
   Url                      = {citeseer.ist.psu.edu/saad93flexible.html}
+}
```

Comment:
I think this was separately merged, as was the next change here, right? Can you rebase to get these out of the way?

Reply:
Will do. I'm just trying to flesh out step-72 before coming back to this. That way step-72 can be reviewed while I make adjustments to this one.